.travis.yml

```diff
@@ -14,3 +14,9 @@ matrix:
     - rvm: jruby-9.1.6.0
 before_install:
   - gem --version
+
+script:
+   - bundle exec rubocop --version
+   - bundle exec rubocop -D .
```

Comment:
Aah, didn't realize you were powering everything with rake.  

As far as parallelization is concerned, I'm genuinely curious why.  It takes far more time to launch the build environment (~30-130 sec, not counting any time to get the spot in the build queue) than it does to run the tests (~2 sec).  Am I missing something?

I'll rebase to drop this change.